packages/stockflux-launcher/public/popupWindow.css

```diff
@@ -0,0 +1,31 @@
+body {
+    overflow: hidden;
+    padding: 4vh 4vw;
+}
+.popup{
+    color:var(--text-grey-lighter);    
+}
+
+.popup-message{
```

Comment:
Could we try the buttons on the same line as the message?

Reply:
![image](https://user-images.githubusercontent.com/49903895/67275098-c4d60c00-f4b9-11e9-99f6-cb9dbf570a96.png)
Example of the message appearing on the same line. This might cause issues with longer string messages?

Reply:
There's a lot of space on the left that could be used, if the text was aligned further to the left side? There should probably also be more space between the buttons and the text.

I don't think it looks significantly better than the way it was before, though, so feel free to change it back if you prefer. Mainly I was thinking we could reduce the amount of empty space at the bottom if it was kept on one line and not two